### ISSUE TITLE
Move Delete Account button in settings

### ISSUE THREAD

**Issue by allywilson** [LemmyNet/lemmy-ui#1775]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

The Delete Account button is very obvious (bright red) and right at the bottom of the Settings page. 

I think it would be better suited to be located elsewhere, I find myself always finding a setting I want to change, then immediately scrolling to the bottom of the page to find/press the Save button - which is usually at the bottom, but in this case it's the Delete Account button and the Save button is above it. I think it would be better if the Delete Account button was moved to below/around where the password reset is located?

**Comment by SineSwiper** [comment]
Agreed.  This is such a dangerous button to put right next to the Save button.

**Comment by Nutomic** [comment]
This was already moved and is not next to save button anymore.
